src/main.rs

```diff
@@ -234,6 +234,13 @@ fn run() -> Result<(), Box<::std::error::Error>> {
                 .value_name("N")
                 .help("Read only N bytes from the input"),
         )
+        .arg(
+            Arg::with_name("c")
```

Comment:
What about using `bytes` as the name for this argument (and the `long(..)` name) in order to be consistent with what `head` and `tail` do (as referenced in the original ticket)?

Reply:
Ok, will change --c to --bytes. Also curious, we are still going to keep this hidden right? Should we add it to the README or just keep it on the secret menu?